src/Microsoft.DotNet.Build.Tasks.Feed/PackageFiles/Microsoft.DotNet.Build.Tasks.Feed.targets

```diff
@@ -234,7 +238,7 @@
       %(...): Metadata is created for each attribute on the element.
     @(OrchestratedBuilds): An item for each Build in the orchestrated build manifest.
```

Comment:
Having both `@(OrchestratedBuilds)` and `@(OrchestratedBuild)` seems bad.  Do we really need both?

Reply:
They don't overlap--`OrchestratedBuild` is the root OrchestratedBuild element, `OrchestratedBuilds` is each Build element within it. Bad similar names though, I'll see what I can do.

Reply:
Maybe just a rename would help here.